KsApi/models/UserCreditCard.swift

```diff
@@ -11,6 +11,15 @@ public struct GraphUserCreditCard: Swift.Decodable {
     public var formattedExpirationDate: String {
       return String(expirationDate.dropLast(3))
     }
+
+    public var imageName: String {
```

Comment:
Extracted this to where it's probably better suited.

Reply:
I like that 👍